tests/integration/basic.spec.js

```diff
@@ -47,6 +47,32 @@ describe('basic', function () {
         expect(response).to.be.an('object').and.to.not.be.empty;
       });
   });
+  it('example of how to set the user-agent', function () {
+    return ogs({
+      url: 'http://ogp.me/',
+      headers: {
+        'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36',
+      },
```

Comment:
I don't see anything unique about this test that shows that the "headers" were actually used.

Reply:
Yes, there is nothing unique about this test. It's not really a test, I just wanted a working example of how you would use headers.